wikibase/README.md

```diff
@@ -81,3 +81,14 @@ Maintenance scripts from extensions and mediawiki core can be run with `docker e
 For example to run a maintenance script from WikibaseImport:
 
 ```docker exec <container name / hash> php //var/www/html/extensions/WikibaseImport/maintenance/importEntities.php --entity Q147```
+
+### Development
+
+A new image should be created for every major release of MediaWiki and Wikibase.
+These images currently use the [MediaWiki docker hub base image](https://hub.docker.com/_/mediawiki), so that needs to have a new version prior to updates here.
+
+ - Create a new release folder, copying the content from a previous release
```

Comment:
Makes sense but really we should probably try and avoid all this copy pasting one day.

Maybe mention something about deleting old releases?

Reply:
Yes, really we should use some form of templating, but I guess this is the current process.
Yes, I'll add a note about removing old releases!